src/common/standardizeUrl.ts

```diff
@@ -0,0 +1,20 @@
+const excludeFromStandardization = ['youtube.com'];
```

Comment:
I think we already have a file that contains all `url` / `link` related functions. Might be a good idea to move this all to there: https://github.com/dailydotdev/daily-api/blob/master/src/common/links.ts